### PR TITLE
feature/print

### PR DESCRIPTION
**Pull request by billyxs**

### CODE REVIEW

File: packages/print/src/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+const Print = ({
+  showIcon,
+  showLabel,
+  label,
+}) => (
+  <Text
+    cursor='pointer'
+    muted
```
[the remark this answers is not in the document]
Comment (billyxs) [reply packages/print/src/index.js]: I updated it but I'm not sure we are aligned on deprecating it.  I would say our ecosystem hasn't been well defined when it comes to what muted and other named states depict, and we've lost the "why" as to many of these ideas.

File: packages/print/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# hixme-ui print
+
+```bash
+npm i --save @hixme-ui/printc
```
Comment (seesleestak) [inline packages/print/README.md]: Extra c

Comment (billyxs) [reply packages/print/README.md]: thx

File: packages/print/src/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (seesleestak) [reply packages/print/src/index.js]: Got it. I don't have an opinion on it, but I wanted to be consistent with previous usage. I think we need a sit-down with Mark on standardizing stuff.

**Pull request merged**